hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -703,7 +703,14 @@ public boolean mkdirs(Path f, FsPermission permission) throws IOException {
 
   @Override
   public long getDefaultBlockSize() {
-    return (long) getConf().getStorageSize(
+    Configuration conf = getConf();
+    ConfigurationSource source;
```

Comment:
NIT: can we wrap this as a helper function as I can see the initialize() has similar logic.